adafruit_bmp280.py

```diff
@@ -502,15 +517,17 @@ class Adafruit_BMP280_SPI(Adafruit_BMP280):
 
     """
 
-    def __init__(self, spi, cs, baudrate=100000):
+    def __init__(
+        self, spi: SPI, cs: Optional[DigitalInOut], baudrate=100000
+    ) -> bytearray:
```

Comment:
Typically __init__()'s return type will be `None` I think that is what we should have here instead of `bytearray`